earth_enterprise/docker/README.md

```diff
@@ -0,0 +1,182 @@
+# Docker Image for Running GE Open Source Earth Server and Fusion Server
+
+## Setting up Docker
+
+### Ubuntu
```

Comment:
Oh, yeah. I don't usually run Cent OS, or RHEL, so I forgot to add instructions for that.

I can run a container and see what you need to run to get Docker set up.